presto-main/src/test/java/io/prestosql/operator/scalar/TestMathFunctions.java

```diff
@@ -688,11 +688,19 @@ public void testRandom()
         functionAssertions.tryEvaluateWithAll("rand()", DOUBLE, TEST_SESSION);
         functionAssertions.tryEvaluateWithAll("random()", DOUBLE, TEST_SESSION);
         functionAssertions.tryEvaluateWithAll("rand(1000)", INTEGER, TEST_SESSION);
+        functionAssertions.tryEvaluateWithAll("random(TINYINT'3', TINYINT'5')", TINYINT, TEST_SESSION);
+        functionAssertions.tryEvaluateWithAll("random(SMALLINT'20000', SMALLINT'30000')", SMALLINT, TEST_SESSION);
+        functionAssertions.tryEvaluateWithAll("random(1000, 2000)", INTEGER, TEST_SESSION);
+        functionAssertions.tryEvaluateWithAll("random(-10, -5", INTEGER, TEST_SESSION);
+        functionAssertions.tryEvaluateWithAll("random(-10, 10", INTEGER, TEST_SESSION);
```

Comment:
The right `)` is missing. Also, could you add negative value cases for TINYINT, SMALLINT and BIGINT?